### DISCUSSION TITLE
Welcome to talk-with-me Discussions!

### DISCUSSION THREAD

**Discussion by ProgrammerIn-wonderland** [immortal-miku/talk-with-me#7] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ProgrammerIn-wonderland** [comment]
https://github.com/ForumPlayer

**Comment by ProgrammerIn-wonderland** [comment]
@ForumPlayer

**Reply by ForumPlayer** [reply]
I'm here now

**Reply by ProgrammerIn-wonderland** [reply]
Hey

**Reply by ForumPlayer** [reply]
Yep, it's also realtime

**Reply by ProgrammerIn-wonderland** [reply]
Woah that's cool

**Reply by ForumPlayer** [reply]
tbh there's a really good reason for the Issues and Discussions being realtime

**Reply by ForumPlayer** [reply]
It can make the conv quicker and more focused, since y'all can just talk about the issue in realtime without needing to switch to an chat platform

**Reply by ForumPlayer** [reply]
well

**Reply by ForumPlayer** [reply]
back to the Matrix
